dpsim-villas/src/InterfaceVillasQueueless.cpp

```diff
@@ -0,0 +1,374 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#include "villas/node/memory.hpp"
+#include "villas/node/memory_type.hpp"
+#include "villas/path.hpp"
+#include "villas/signal_type.hpp"
+#include <dpsim-villas/InterfaceVillasQueueless.h>
+#include <dpsim-villas/InterfaceWorkerVillas.h>
+#include <memory>
+#include <spdlog/spdlog.h>
+#include <typeinfo>
+#include <villas/node.h>
+
+using namespace CPS;
+using namespace villas;
+
+namespace DPsim {
+
+InterfaceVillasQueueless::InterfaceVillasQueueless(
```

Comment:
What about naming the interface directly about what they do?

E.g.

- VillasInterfaceBuffered
- VillasInterfaceQueued

"Queueless" is not descriptive here.

Reply:
It's not buffered though. I think something like "direct" is not descriptive either. My reasoning is that the main property of the interface is that it does not use a queue and therefore needs to be used with quite a bit more caution.